rest_framework/views.py

```diff
@@ -356,7 +356,17 @@ def check_throttles(self, request):
                 throttle_durations.append(throttle.wait())
 
         if throttle_durations:
-            self.throttled(request, max(throttle_durations))
+            # Filter out `None` values which may happen in case of config / rate
+            # changes, see #1438
+            durations = [
+                duration for duration in throttle_durations
+                if duration is not None
+            ]
+
+            if durations:
+                self.throttled(request, max(durations))
+            else:
+                self.throttled(request, None)
```

Comment:
I might change this to a ternary, but either way is 👌

```python
duration = max(durations) if durations else None
self.throttled(request, duration)
```

Reply:
Changed it in 957b39fa